### PR TITLE
Add Personal Page - Marsian

### PR DESCRIPTION
**Pull request by marsian83**
## Description

Add Personal Page for Marsian
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

Your ENS/address: `0x472cAde746560419a407CA7103c87c79c794C418`

### CODE REVIEW

File: packages/nextjs/app/builders/0x472cAde746560419a407CA7103c87c79c794C418/page.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          {">"} may or may not be from the planet Mars
+        </p>
+
+        <Address address="0x472cAde746560419a407CA7103c87c79c794C418" format="long" />
+
+        <div className="flex items-center gap-x-6">
+          <h2 className="pt-2 font-semibold text-teal-300 text-lg">Connect : </h2>
+
+          {socials.map((social, key) => (
+            <Link
```
Comment (carletex) [inline packages/nextjs/app/builders/0x472cAde746560419a407CA7103c87c79c794C418/page.tsx:45]: Let's better use `<a>` for these. `<Link>` is meant for internal / client-side navigation (with nice features as prefetch, etc)

**Review by carletex — CHANGES_REQUESTED** [review]
This looks awesome @marsian83 !! Love the animation <3

Just left a minor review comment. After that, ready to merge

File: packages/nextjs/app/builders/0x472cAde746560419a407CA7103c87c79c794C418/page.tsx (hunk 1 above)
Comment (marsian83) [reply packages/nextjs/app/builders/0x472cAde746560419a407CA7103c87c79c794C418/page.tsx]: Hi @carletex, thanks for the review
I have changed it

**Comment by carletex** [comment]
Good stuff!!

**Pull request merged**